components/viz/service/BUILD.gn

```diff
@@ -341,6 +341,16 @@ viz_component("service") {
     ]
   }
 
+  if (is_cobalt) {
```

Comment:
I think here you want to use use_starboard_media instead of is_cobalt

Reply:
`use_starboard_media` is not globally defined, it can be only used when we include `//starboard/build:starboard_buildflags`.

Reply:
Similar to how you do the C++ guarding, you would import `//starboard/build/buildflags.gni` at the top of the file:

```
if (is_cobalt) {
  import("//starboard/build/buildflags.gni")
}

...

if (is_cobalt && use_starboard_media) {
  // include media sources
}
```
Like the C++ guarding, I also hope to make this easier

Reply:
Fixed.